corgi/web/templates/product_manifest.json

```diff
@@ -9,7 +9,7 @@
     {% endfor %}
     {
         "copyrightText": "NOASSERTION",
-        "downloadLocation": {% if obj.channels %}"{{obj.channels.0}}"{% else %}"NOASSERTION"{% endif %},
```

Comment:
A better download location is probably [package browser](https://access.redhat.com/downloads/content/package-browser) for (S)RPMs and Modules, and [catalog](https://catalog.redhat.com/software/containers/search) for Containers.

Reply:
Thank you! I was mostly just guessing when I decided to use channels[0] originally (didn't even really know what a channel was back then). I saw while tweaking other code that we had a nice download_url property, and didn't realize it was internal. I'll switch this to use above.